pkg/engine2/operational_eval/vertex_property.go

```diff
@@ -119,12 +119,20 @@ func (v *propertyVertex) Evaluate(eval *Evaluator) error {
 		return err
 	}
 
+	// we know we cannot change properties of imported resources only users through constraints
+	// we still want to be able to update ids in case they are setting the property of a namespaced resource
+	// so we just conditionally run the edge operational rules
+	//
+	// we still need to run the resource operational rules though,
+	// to make sure dependencies exist where properties have operational rules set
```

Comment:
I still think this isn't the right approach to importing, but I won't block the PR on it.

Reply:
vpc is a simpler case so ill try to add another imported resource and see if this falls apart, but i do think it makes sense to have our logic account for this rather than a special case in the beginning because we do need to use rules to determine edges, etc